app/components/header/header.component.html

```diff
@@ -427,29 +433,3 @@ <h4 class="modal-title">Confirmation</h4>
   </div>
 </div>
 
-<!--Modal for adding stash message -->
```

Comment:
You probably don't want to delete this!!!

Reply:
It created a merge conflict so I removed it initially , I believe it was not the issue-70 branch but somehow got included during a pull update. However I have updated the file and re-introduced the code.